images/bootstrap/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMG = gcr.io/k8s-testimages/bootstrap
```

Comment:
I'm late to the party, but this name is a bit misleading, since it has nothing to do with the test-infra bootstrap. something like "kubetest-base" or something might have been a bit better.

Reply:
I think we chose this name with the intention that it has all of the *requirements* for bootstrap. In the future it should also have bootstrap in it. We should probably add `README`s explaining these @krzyzacy

Reply:
yeah, I'll probably also move kubekins out of Jenkins